scalalib/src/mill/scalalib/ScalaModule.scala

```diff
@@ -62,8 +61,12 @@ trait ScalaModule extends JavaModule { outer =>
 
   def scalacPluginIvyDeps = T{ Agg.empty[Dep] }
 
+  def scalaDocPluginIvyDeps = scalacPluginIvyDeps
+
   def scalacOptions = T{ Seq.empty[String] }
 
+  def scalaDocOptions = scalacOptions
```

Comment:
This (and above) should probably be `T{ scalacOptions() }` for consistency with the rest of the code

Reply:
Done :+1: